src/webgl/material.js

```diff
@@ -128,10 +134,21 @@ p5.prototype.loadShader = function(
  * let varying = 'precision highp float; varying vec2 vPos;';
  *
  * // the vertex shader is called for each vertex
- * let vs =
- *   varying +
- *   'attribute vec3 aPosition;' +
- *   'void main() { vPos = (gl_Position = vec4(aPosition,1.0)).xy; }';
+ * let vs = `
+ * uniform mat4 uModelViewMatrix;
+ * uniform mat4 uProjectionMatrix;
+ *
+ * attribute vec3 aPosition;
+ * attribute vec2 aTexCoord;
+ * varying vec2 vTexCoord;
+ *
+ * void main() {
+ *   vTexCoord = aTexCoord;
+ *   vec4 positionVec4 = vec4(aPosition, 1.0);
+ *   gl_Position = uProjectionMatrix * uModelViewMatrix * positionVec4;
+ *  }
+ * `;
+ *
  *
  * // the fragment shader is called for each pixel
  * let fs =
```

Comment:
- Let's add `precision highp float;` to the top of this one too
- We can take out the `varying +` line
- A few lines below where it uses `vPos`, we can just replace that with `gl_FragCoord.xy`
- Maybe let's also convert this shader to use one big backtick string instead of a bunch of strings added together?